README.md

```diff
@@ -14,24 +14,32 @@ The code is available in the `src/` directory.
-- **evaluation_split_point**: define training and testing spliting point in the dataset, for model evaluation during learning phase (fit takes twice as long time)
-- **prediction_horizons**: list of prediction horizons (in hours) for which the model will be trained to predict for.
-- **sesnors**: list of sensors for which this specific instance will train the models and will be making predictions.
-- **retrain_period**: A number of recieved samples after which the model will be re-trained. This is an optional parameter. If it is not specified no re-training will be done.
-- **samples_for_retrain**: A number of samples that will be used for re-training. If retrain_period is not specified this parameter will be ignored. This is an optional parameter. If it is not specified (and retrain_period is) the re-train will be done on all samples recieved since the component was started.
 
+| Name | Type | Default | Description |
+| --- | --- | --- | --- |
+| **prediction_horizons**| list(integer) | | List of prediction horizons (in hours) for which the model will be trained to predict for.|
+| **sensors**| list(string) | | List of sensors for which this specific instance will train the models and will be making predictions.|
+| **bootstrap_servers**| string or list(string)|  | String (or list of `host[:port]` strings) that the consumer should contact to bootstrap initial cluster metadata.|
+| **algorithm**| string | `torch` | String as either a scikit-learn model constructor with initialization parameters or a string `torch` to train using a pre defined neural network using PyTorch with architecture: \[torch.nn.Linear, torch.nn.ReLU, torch.nn.Linear\],|
+| **evaluation_period**| integer | 512 | Define time period (in defined time offset that is hours by default) for which the model will be evaluated during live predictions (evaluations metrics added to output record).|
+| **evaluation_split_point**| float | 0.8 | Define training and testing splitting point in the dataset, for model evaluation during learning phase (fit takes twice as long time).|
+| **retrain_period**| integer | None | A number of received samples after which the model will be re-trained. This is an optional parameter. If it is not specified no re-training will be done.|
+| **samples_for_retrain**| integer | None | A number of samples that will be used for re-training. If retrain_period is not specified this parameter will be ignored. This is an optional parameter. If it is not specified (and retrain_period is) the re-train will be done on all samples received since the component was started.|
+| **time_offset**| string | H | [String alias](https://pandas.pydata.org/pandas-docs/stable/user_guide/timeseries.html#offset-aliases) to define the data time offsets. The aliases used in training and topic names are lowercase for backwards compatibility.|
```

Comment:
This is not very clear to me since it is new, does it affect ScikitLearn models?

Reply:
Is this just the unit for prediction_horizons? If so - move these descriptions closer together.

Reply:
Yes. It is as you said a unit for prediction_horizons. I will also write about time_offset in prediction_horizons and move them together. 

The sk-learn models can now also use different time offsets but if nothing is specified it defaults to hours for backwards compatibility.